### PR TITLE
MWPW-148256 decorate catalog cards & some sidenav with analytics attributes

### PR DESCRIPTION
**Pull request by npeltier**
Resolves: [MWPW-148256](https://jira.corp.adobe.com/browse/MWPW-148256)

**Test URLs:**
- Before: https://main--cc--adobecom.hlx.live/products/illustrator?martech=off&mep=off&georouting=off
- After: https://MWPW-148256--cc--npeltier.hlx.live/products/illustrator?martech=off&georouting=off&mep=off

- Before: https://main--cc--adobecom.hlx.live/products/catalog?martech=off&georouting=off&mep=off
- After: https://MWPW-148256--cc--npeltier.hlx.live/products/catalog?martech=off&georouting=off&mep=off

### CODE REVIEW

File: creativecloud/blocks/catalog/catalog.js [hunk 1]
```diff
@@ -25,12 +25,28 @@ export default async function init(el) {
       const { default: initSidenav } = await import('../sidenav/sidenav.js');
       const sidenav = await initSidenav(sidenavEl);
       el.appendChild(sidenav);
+      sidenav.setAttribute('daa-lh', 'b1|catalog-sidenav');
```
Comment (yesil) [inline creativecloud/blocks/catalog/catalog.js:28]: b1 is usually the marquee though.
should we assign the number of block manually like this?

**Review by yesil — COMMENTED** [review]
Besides assigning the block number manually, looks good to me.
I'll wait the end of discussions before ✅

File: creativecloud/blocks/catalog/catalog.js (hunk 1 above)
Comment (npeltier) [reply creativecloud/blocks/catalog/catalog.js:28]: bX is the number within the section, and this one is (and will be) the first always per catalog component layout

**Comment by Roycethan** [comment]
@npeltier  Checked on your link and following are my findings, plz review thanks

1. Web interaction name for Link to special offers in the side nav is : “RESOURCES--resources|s2|b1|catalog|b1|catalog-sidenav” - It does not contain any mention of link name “special-offers’
2. No event triggered for “Search all products”
3. No events triggered for “Sort:Popularity” or “Sort:alphabetically”
4. No events triggered for Action menu which user can click to see device types on the hovering over each product card.
5. On clicking out side the TWP modal(free-trail/buy now) user can close the modal- But in this case event is not triggered.

**Comment by npeltier** [comment]
as discussed @Roycethan deferred sort + search events to [MWPW-148256](https://jira.corp.adobe.com/browse/MWPW-148256)](https://jira.corp.adobe.com/browse/MWPW-153245)

**Comment by Roycethan** [comment]
> @npeltier Checked on your link and following are my findings, plz review thanks
> 
> 1. Web interaction name for Link to special offers in the side nav is : “RESOURCES--resources|s2|b1|catalog|b1|catalog-sidenav” - It does not contain any mention of link name “special-offers’
> 2. No event triggered for “Search all products”
> 3. No events triggered for “Sort:Popularity” or “Sort:alphabetically”
> 4. No events triggered for Action menu which user can click to see device types on the hovering over each product card.
> 5. On clicking out side the TWP modal(free-trail/buy now) user can close the modal- But in this case event is not triggered.

@npeltier  
On latest:
1) has this link name
"RESOURCES--special-offers|s2|b1|catalog|nopzn|catalog-devicet|b1|catalog-sidenav"
good part is : it have special-offers
I didn't get what stands for " |nopzn|catalog-devicet " in above link name ?

2,3,4,5 are still open - ad have mentioned these in https://jira.corp.adobe.com/browse/MWPW-153245

**Review by yesil — APPROVED** [review]

File: creativecloud/blocks/sidenav/sidenav.js [hunk 2]
```diff
@@ -9,22 +9,39 @@ const makePause = async (timeout = 0) => new Promise((resolve) => setTimeout(res
 
 const getIdLeaf = (id) => (id?.substring(id.lastIndexOf('/') + 1) || id).toLowerCase();
 
+const computeDaaLLText = (text) => {
+  const tokens = text.split('-');
```
Comment (3ch023) [inline creativecloud/blocks/sidenav/sidenav.js:13]: text?.split('-');

**Review by 3ch023 — APPROVED** [review]

**Review by salonijain3 — APPROVED** [review]

**Pull request merged**